.CI/update_all_tokens.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function
+
+import os
+import requests
+
+import ruamel.yaml
+import subprocess
+from github import Github
+
+circle_token = os.environ["CIRCLE_TOKEN"]
+appveyor_token = os.environ["APPVEYOR_TOKEN"]
+anaconda_token = os.environ["BINSTAR_TOKEN"]
+gh_token = os.environ["GH_TOKEN"]
+
+smithy_conf = os.path.expanduser('~/.conda-smithy')
+if not os.path.exists(smithy_conf):
+    os.mkdir(smithy_conf)
+
+def write_token(name, token):
+    with open(os.path.join(smithy_conf, name + '.token'), 'w') as fh:
+        fh.write(token)
+
+if 'APPVEYOR_TOKEN' in os.environ:
+    write_token('appveyor', os.environ['APPVEYOR_TOKEN'])
+if 'CIRCLE_TOKEN' in os.environ:
+    write_token('circle', os.environ['CIRCLE_TOKEN'])
+if 'GH_TOKEN' in os.environ:
+    write_token('github', os.environ['GH_TOKEN'])
+
+from conda_smithy.ci_register import (add_token_to_circle, travis_headers,
+    travis_encrypt_binstar_token, appveyor_encrypt_binstar_token,
+    travis_token_update_conda_forge_config)
```

Comment:
It's weird that with AppVeyor we join these into one step (encrypting the token and adding it to the config file), but Travis CI we do not. Nothing we can do about it here for now, but maybe we should pick one of these approaches.